controller/UserController.js

```diff
@@ -0,0 +1,21 @@
+import passwordHash from 'password-hash';
+
+
+class UserController {
+    signUp(req, res) {
+        const user = {
+            firstName: req.body.firstName,
+            lastName: req.body.lastName,
+            email: req.body.email,
+            passwordHash: passwordHash.generate(req.body.password),
+            password_confirmation: passwordHash.generate(req.body.password),
+        }
+
+        if (!user.firstName || !user.email) {
```

Comment:
Why are you validating in your controller? Validation should happen somewhere else as a middleware